docs/src/main/sphinx/installation/cli.rst

```diff
@@ -405,6 +418,23 @@ Press :kbd:`→` to accept the suggestion and replace the current command line
 buffer. Press :kbd:`Ctrl+→` (:kbd:`Option+→` on Mac) to accept only the next
 keyword. Continue typing to reject the suggestion.
 
+Configuration file
+------------------
+
+The CLI can read default values for all options from a file. It uses the first
+file found from the ordered list of locations:
+* File path set as value of the ``TRINO_CONFIG`` environment variable.
+* ``.trino_config`` in the current users home directory.
+* ``$XDG_CONFIG_HOME/trino/config``.
+
+Use option names as keys, without the ``--`` prefix.
```

Comment:
Maybe reword to something like

You can use all supported options without the -- prefix in the configuration properties file, as displayed in the following example:

Reply:
Thanks!